ambry-quota/src/main/java/com/github/ambry/quota/AmbryQuotaManager.java

```diff
@@ -121,16 +121,38 @@ public ThrottlingRecommendation getThrottleRecommendation(RestRequest restReques
     return throttlingRecommendation;
   }
 
+  @Override
+  public boolean isQuotaExceedAllowed(RestRequest restRequest) {
```

Comment:
how are we going to use this method? when we do 
```
if (quotaManager.getThrottleRecommend().shouldThrottle()) {
    if (quotaManager.isQuotaExceedAllowed()) {
    xxxx
    }
}
```
?
Is this how we are going to use this method in the end?